canopas/vue-file-upload
# Added live demo in readme

> PR Description
- Add a full video of the library at the top of the `README.md` file
- Added link of code-sandbox live demo of the library in the `README.md` file
- Refactor code to consider files after uploading

> Test Steps
- Checkout the readme file https://github.com/canopas/vue-file-upload/tree/add-live-demo-in-readme#file-management-with-preview
- Click on the icon to see the live demo
